flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/CheckpointCoordinator.java

```diff
@@ -181,6 +181,9 @@
 	/** Registry that tracks state which is shared across (incremental) checkpoints */
 	private SharedStateRegistry sharedStateRegistry;
 
+	/** Determines if a tasks are failed or not if there is an error in their checkpointing. */
```

Comment:
"a tasks are" should be updated. 
And should description be aligned to `periodicScheduling` and `triggerRequestQueued`

other places should also be updated.